mantidproject/mantid
# Use Mslice as conda package instead of an external interface

### Description of work

#### Summary of work
MSlice is now installed via conda and not as an external interface anymore.

Fixes #38215.

### To test:

Verify that in Mantid workbench, the MSlice interface can still be opened. De-install the `mslice` package from a conda environment with the workbench package installed. When you now try to open the MSlice interface, there should be a warning 'Mslice is not available', but no crash. In a mantid-developer environment, `mslice` should not be available without being manually installed by the user. Again, there should be a warning 'Mslice is not available', but no crash.

The packages can be found here: https://builds.mantidproject.org/job/build_packages_from_branch/944/

*This does not require release notes* because **this is not a change that is visible to the user.**

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
